internal/action/sync.go

```diff
@@ -181,11 +195,12 @@ func (s *Action) syncMount(ctx context.Context, mp string) error {
 	}
 	syncPrintDiff(ctxno, l, ln)
 
-	debug.Log("Syncing Mount %s. Exportkeys: %t", mp, ctxutil.IsExportKeys(ctx))
+	exportKeys := s.cfg.GetBool("core.exportkeys")
```

Comment:
I see we're using both `s.cfg` and `config.FromContext` in this file, I'm not sure I grasp fully the difference, if it's on purpose or if we could remove the `cfg` from the `Action` struct since it's now in the contexts?

Reply:
No. I think we should use s.cfg when it's available. The FromContext was a mistake.